dotCMS/src/main/java/com/liferay/portal/model/User.java

```diff
@@ -300,11 +300,10 @@ public void setModificationDate(final Date modificationDate) {
         setModified(true);
     }
 
-  public boolean isAnonymousUser(){
+  	public boolean isAnonymousUser(){

```

Comment:
This class only has blank spaces. Consider reverting it.

Reply:
javadoc also https://github.com/dotCMS/core/commit/53df03a0798348247b767f3b45da91c6ed66acea#diff-c944100d5305df9ed32bfd3b81e7ca81R307